stories/byoc.stories.js

```diff
@@ -1,15 +1,6 @@
-import moment from 'moment';
```

Comment:
Just a comment. In all stories I changed that the time parameters now use `Date`, not moment.